shared/feature/profile/src/main/java/com/epmedu/animeal/profile/presentation/ui/ProfileInputForm.kt

```diff
@@ -22,15 +33,20 @@ import com.epmedu.animeal.profile.presentation.ProfileInputFormEvent.SurnameChan
 import com.epmedu.animeal.profile.presentation.viewmodel.ProfileState
 import com.epmedu.animeal.profile.presentation.viewmodel.ProfileState.FormState.READ_ONLY
 import com.epmedu.animeal.resources.R
+import kotlinx.coroutines.CoroutineScope
+import kotlinx.coroutines.launch
 
 @Composable
 fun ProfileInputForm(
+    modifier: Modifier = Modifier,
```

Comment:
According to [Google Guidelines](https://android.googlesource.com/platform/frameworks/support/+/androidx-main/compose/docs/compose-api-guidelines.md#elements-accept-and-respect-a-modifier-parameter) modifier must appear as the first **optional** parameter in the element function's parameter list. Required parameters must appear before it.